packages/nx/src/command-line/migrate.ts

```diff
@@ -1010,7 +1016,7 @@ export async function executeMigrations(
 
 async function runMigrations(
   root: string,
-  opts: { runMigrations: string },
+  opts: { runMigrations: string; ifExists: boolean },
```

Comment:
This option needs to be added into the nx-commands.ts file as well for the migrate command so that it is generated in our documenation and `--help` text.

Reply:
@AgentEnder Sure I'll do it. Is there a way to let the pipeline run? I'm having problems running it locally in Windows

Reply:
I've only seen the issue below on one or two others, and unfortunately I have no idea what causes it. I'll push up a commit to the branch, which generally lets CI through for new contributors, but its not a guarantee.